### PR TITLE
HIPP-244: Scopes granted success page

### PR DESCRIPTION
**Pull request by PaulCDurham**
https://jira.tools.tax.service.gov.uk/browse/HIPP-244

### CODE REVIEW

File: conf/messages.en [hunk 1]
```diff
@@ -125,3 +125,8 @@ addScope.error.length = Scope name must be 100 characters or less
 
 pendingApprovals.title = Pending production approvals
 pendingApprovals.heading = Pending production approvals
+
+scopeApproved.title = Scope approved successfully
+scopeApproved.heading = Scope approved successfully
+scopeApproved.guidance.heading = What you can do now
+scopeApproved.guidance.content = Return to {0}.
```
Comment (laurahk) [inline conf/messages.en:132]: is this syntax `{0}` to do with navigation? Is a play thing?

**Review by laurahk — APPROVED** [review]

**Comment by laurahk** [comment]
Only thing I noticed was an extra space on the link
link-has-space

**Comment by laurahk** [comment]
Looks good to me. Thanks for the comment on the ticket about the other tickets need to be able to test the ACs.

File: conf/messages.en (hunk 1 above)
Comment (PaulCDurham) [reply conf/messages.en:132]: It's a placeholder. You can call messages("page.text", placeholder) to use string substitution. In this case we want a link there so we want to substitute in the <a> element.

**Pull request merged**